specification/batch/resource-manager/Microsoft.Batch/stable/2019-04-01/BatchManagement.json

```diff
@@ -2016,14 +2016,14 @@
           "x-nullable": false,
           "type": "integer",
           "format": "int32",
-          "title": "The dedicated core quota for the Batch account."
+          "title": "The dedicated core quota for the Batch account. For accounts with PoolAllocationMode set to UserSubscription, quota is managed on the subscription so this value is null."
```

Comment:
Do we say `null` other places in the spec?

I think we prefer "This value is not returned" - because `null`  has specific language implications.

Reply:
We do not. I changed to not returned as that more accurately describes the behavior of the rest api anyways. These values are not a part of the response if PoolAllocationMode is set to UserSubscription.